extensions/amp-lightbox/1.0/test-e2e/test-custom-close-button.js

```diff
@@ -45,7 +45,8 @@ describes.endtoend(
       ).to.equal(0);
     });
 
-    it('should open the lightbox', async () => {
+    // TODO(wg-components, #28948): Flaky during CI.
+    it.skip('should open the lightbox', async () => {
```

Comment:
This test is still red on `master`. Looks like the failure source is in `extensions/amp-lightbox/test-e2e/test-amp-lightbox.js`.